app/models/competition.rb

```diff
@@ -242,6 +246,24 @@ class Competition < ApplicationRecord
   validates :external_website, :external_registration_page, length: { maximum: MAX_URL_LENGTH }
   validates :contact, length: { maximum: MAX_MARKDOWN_LENGTH }
 
+  validate :validate_newcomer_reserved_spots
+  private def validate_newcomer_reserved_spots
+    return unless competitor_limit.present?
+
+    max_newcomer_spots = (competitor_limit * MAX_NEWCOMER_SPOTS_RESERVED_FRACTION).to_i
```

Comment:
Do the rules of Newcomer Month (or WCRP or whatnot) somehow indicate whether this should be rounded up or down?
In any case, I think it would be good to make that explicit by using either `floor` or `ceil` (or perhaps even `round` FWIW)

Reply:
It doesn't - I've set it to `floor`, on the assumptions that (a) it doesn't matter, and (b) if it did, we'd rather have 1 more flexible spot than 1 specifically reserved for newcomers - given that the newcomer reservation portion is already very generous